### PR TITLE
Ocrvs 10612

### LINKED ISSUE #10612: 'Age' is not shown on Verfiy Identity page in printing flow for death events.
**Bug description:**
When 'exact dob unknown' is set 'yes' for informant/ spouse, 'Age' is not shown on the Verify identity page in printing flow for death events.

**Steps to produce:**

1. Navigate to the printing flow of any death record that has age for informant/ spouse.
2. Navigate to the Verify identity page.
3. Observe that 'age' is not shown on that page.

**Actual result:**
When 'exact dob unknown' is set 'yes' for informant/ spouse, 'Age' is not shown on the Verify identity page in printing flow for death events.

**Expected result:**
'Age' should be shown on the Verify identity page in printing flow for death events.

**Screenshots:**

For informant -
Image
Image

For spouse -
Image
Image

**Tested on:**
https://register.v19-beta-prod.opencrvs.dev/

### PR DESCRIPTION
**Pull request by makelicious**
## Description

Clearly describe what has been changed. Include relevant context or background.
Explain how the issue was fixed (if applicable) and the root cause.

Link this pull request to the GitHub issue (and optionally name the branch `ocrvs-<issue #>`)

## Checklist

- [ ] I have linked the correct Github issue under "Development"
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

**Comment by makelicious** [comment]
https://github.com/opencrvs/e2e/actions/runs/18405430207

File: e2e/testcases/v2-test-data/death-declaration.ts [hunk 1]
```diff
@@ -64,6 +65,7 @@ export async function getDeclaration({
 
   const mockDeclaration = {
     'spouse.dob': '1975-02-18',
+    'spouse.age': undefined,
```
Comment (Zangetsu101) [inline e2e/testcases/v2-test-data/death-declaration.ts:68]: Feels a bit weird that it's needed explicitly

**Review by Zangetsu101 — APPROVED** [review]

File: e2e/testcases/v2-test-data/death-declaration.ts (hunk 1 above)
Comment (makelicious) [reply e2e/testcases/v2-test-data/death-declaration.ts:68]: Ah true. It does not need it, it is stripped out during payload. It just helps with the autocomplete on tests.
